sklearn/preprocessing/tests/test_label.py

```diff
@@ -196,11 +196,6 @@ def test_label_encoder_fit_transform():
 
 
 def test_label_encoder_errors():
-    # Check that invalid arguments yield ValueError
-    le = LabelEncoder()
-    assert_raises(ValueError, le.transform, [])
```

Comment:
Why doing so?

Reply:
Because right now we're returning [] when the  `_num_samples == 0`

Reply:
I don't think it's right. You should move the check after check_is_fitted

Reply:
Alright, done